deployment/confStorage/download.py

```diff
@@ -73,6 +73,11 @@ def download_cluster_configuration(self, local_path):
             self.logger.error("The configuration doesn't exists on your cluster. Please upload it first.")
             sys.exit(1)
 
+        if u'config.yaml' not in configuration_dict:
```

Comment:
typo ? `u'config.yaml'`

Reply:
Keys in `configuration_dict` are unicode